python-sdk/src/astro/databricks/databricks_api_utils.py

```diff
@@ -0,0 +1,144 @@
+import fileinput
+import pathlib
+import time
+from pathlib import Path
+from typing import Dict
+
+from attrs import define, field
+from databricks_cli.dbfs.api import DbfsApi, DbfsPath
+from databricks_cli.jobs.api import JobsApi
+from databricks_cli.runs.api import RunsApi
+from databricks_cli.sdk.api_client import ApiClient
+
+from astro.databricks.load_file.load_file_python_code_generator import render
+
+cwd = pathlib.Path(__file__).parent
+
+
+@define
+class DeltaOption:
+    """
+    Helper class for rendering options into COPY_INTO Databricks SQL statements
+    """
+
+    option_dict: dict = field(factory=dict)
+
+    def empty(self):
+        return len(self.option_dict or {}) == 0
```

Comment:
`self.option_dict` is by default `{}` so we don't need the `or {}` part.


```python
In [3]: from attrs import define, field

In [4]: @define
   ...: class DeltaOption:
   ...:     """
   ...:     Helper class for rendering options into COPY_INTO Databricks SQL statements
   ...:     """
   ...:
   ...:     option_dict: dict = field(factory=dict)
   ...:

In [5]: DeltaOption().option_dict
Out[5]: {}
```

Reply:
I was running into issues where option_dict was None. I'll try it again and see if I get the same issue.